docs/gl_objects/iterations.rst

```diff
@@ -0,0 +1,33 @@
+##########
+Iterations
+##########
+
+
+Reference
+---------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.GroupIteration`
+  + :class:`gitlab.v4.objects.GroupIterationManager`
+  + :attr:`gitlab.v4.objects.Group.iterations`
+  + :class:`gitlab.v4.objects.ProjectIterationManager`
+  + :attr:`gitlab.v4.objects.Project.iterations`
+
+* GitLab API: https://docs.gitlab.com/ee/api/iterations.html
+
+Examples
+--------
+
+.. note::
+
+    GitLab no longer has project iterations. Using a project endpoint returns
+    the ancestor groups' iterations. 
+
+List iterations for a project's ancestor groups::
+
+    iterations = project.iterations.list()
+
+List iterations for a group::
+
+    iterations = project.iterations.list()
```

Comment:
I don't understand the difference between these two examples 😕

Reply:
Ah, copy/paste error! Thanks

Reply:
Changed in https://github.com/python-gitlab/python-gitlab/commit/2de65b36cc2ade168b75af4618afa96bf78b9f45.